### PR TITLE
CORTX-29982: incorrect list-item-spacing

### PR DESCRIPTION
**Pull request by hessio**
CORTX-29982: Incorrect list-item spacing

Solution: Add newline at end of each list item.

Signed-off-by: hessio patrick.hession@seagate.com

-----
[View rendered CODE_OF_CONDUCT.md](https://github.com/hessio/cortx-hare/blob/list-looseness/CODE_OF_CONDUCT.md)
[View rendered README.md](https://github.com/hessio/cortx-hare/blob/list-looseness/README.md)
[View rendered rfc/11/README.md](https://github.com/hessio/cortx-hare/blob/list-looseness/rfc/11/README.md)
[View rendered rfc/13/README.md](https://github.com/hessio/cortx-hare/blob/list-looseness/rfc/13/README.md)
[View rendered rfc/16/README.md](https://github.com/hessio/cortx-hare/blob/list-looseness/rfc/16/README.md)
[View rendered rfc/19/README.md](https://github.com/hessio/cortx-hare/blob/list-looseness/rfc/19/README.md)
[View rendered rfc/20/README.md](https://github.com/hessio/cortx-hare/blob/list-looseness/rfc/20/README.md)
[View rendered rfc/21/README.md](https://github.com/hessio/cortx-hare/blob/list-looseness/rfc/21/README.md)

### CODE REVIEW

File: cfgen/cfgen [hunk 1]
```diff
@@ -1284,6 +1284,8 @@ def service_types(proc_t: ProcT,
             ts.extend([SvcT.M0_CST_IOS,
                        SvcT.M0_CST_SNS_REP,
                        SvcT.M0_CST_SNS_REB,
+                       SvcT.M0_CST_DIX_REP,
```
Comment (mssawant) [inline cfgen/cfgen:1287]: Sorry, I don't understand the codacy error in this case.

File: hax/hax/ha/__init__.py [hunk 2]
```diff
@@ -49,22 +47,19 @@ def create_ha_thread(planner: WorkPlanner,
                      util: ConsulUtil) -> StoppableThread:
     """Creates the proper HA-aware thread, handling possible import errors."""
     try:
-        from .thread import EventPollingThread
-        _ha_message_bus_init(util)
```
Comment (mssawant) [inline hax/hax/ha/__init__.py:53]: Where is this invoked now?

**Review by mssawant — COMMENTED** [review]
@hessio, I see lot of code changes which some how doesn't seem to be related. Did you rebase your branch with the latest Hare main?

**Comment by mssawant** [comment]
@hessio, I am trying to rebase but getting conflicts, can you submit a new patch with just your changes?

**Comment by hessio** [comment]
> @hessio, I am trying to rebase but getting conflicts, can you submit a new patch with just your changes?

Ok no problem

**Pull request closed without merging**